keylime/da/examples/file.py

```diff
@@ -51,15 +53,34 @@ def bulk_record_retrieval(self, record_identifier):
         record_list = []
 
         with open(record_identifier, "rb") as fp:
+
+            if self.only_last_record_wanted(start_date, end_date):
+                start_date = 0
+                try:
+                    fp.seek(-2, os.SEEK_END)
+                    while fp.read(1) != b"\n":
+                        fp.seek(-2, os.SEEK_CUR)
+
+                    fp.seek(-2, os.SEEK_CUR)
+                    while fp.read(1) != b"\n":
+                        fp.seek(-2, os.SEEK_CUR)
```

Comment:
I do not get this logic. What are we looking for?

Reply:
the penultimate line of the line (the last line is just a separator)

Reply:
Can you include a comment about? Answering the "why" is a good reason for a comment.

Reply:
fixed